lib/tom_queue/worker_supervisor.rb

```diff
@@ -0,0 +1,66 @@
+module TomQueue
```

Comment:
requirey things needed here

Reply:
Potentially not if you use the pattern I allude to in another comment